defaults/main.yml

```diff
@@ -14,3 +14,7 @@ r_packages_repos: "{{ r_cran_mirror }}"
 r_packages_lib: /usr/local/lib/R/site-library
 r_packages: []
 r_environment: {}
+
+r_scripts_install_path: /usr/local/bin
+r_scripts_install_owner: "{{ ansible_user_id | default('root') }}"
+r_scripts_install_group: "{{ r_scripts_install_owner | default('root') }}"
```

Comment:
Sorry for the confusion. I meant that you could use `ansible_user_id` to override `r_scripts_install_owner`. But for this implementation I prefer:

```yml
r_scripts_install_owner: root
r_scripts_install_group: "{{ r_scripts_install_owner }}"
```

Reply:
I think a dynamic value is better for this. As it will work for every use case.

So I don't understand why you prefer to have the owner hard coded here. 

Please could you explain ?

Reply:
Because it's "more" backwards compatible and "less magic". I (really) understand your use case, but this role was never intended to be used that way and I don't think people do so.

Reply:
Maybe add and example to the `README` on how to configure "your use case" and which variables to set / override.

Reply:
Done:https://github.com/Oefenweb/ansible-r/pull/52/commits/c6a526871a1079e8442cd685fbb6c46aacd6d9fe and https://github.com/Oefenweb/ansible-r/pull/52/commits/60c79cf420cc12d5a000687a810784ec172e74c0